lib/internal/util.js

```diff
@@ -120,7 +120,7 @@ function decorateErrorStack(err, options) {
     err.message = `The requested module '${childSpecifier}' is expected ` +
       'to be of type CommonJS, which does not support named exports. ' +
       'CommonJS modules can be imported by importing the default export. ' +
-      'The following code will work:\n' +
+      'For example:\n' +
       `import pkg from '${childSpecifier}';\n` +
       `const ${namedImports} = pkg;`;
```

Comment:
Can you paste an example of the output when you test this?

Are you going to add a test?

Reply:
Working on tests right now, will push something before I go to bed